docs/reference/method/MongoDBDatabase-aggregate.txt

```diff
@@ -0,0 +1,71 @@
+==============================
+MongoDB\\Database::aggregate()
+==============================
+
+.. versionadded:: 1.5
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Database::aggregate()
+
+   Executes an :manual:`aggregation framework pipeline
+   </core/aggregation-pipeline>` operation on the collection.
```

Comment:
"on the collection" is copypasta, but we don't want to say "on the database" as that also isn't accurate. Let's borrow language from [`db.aggregate()`](https://docs.mongodb.com/manual/reference/method/db.aggregate/).

```
Runs a specified :manual:`admin/diagnostic pipeline </reference/operator/aggregation-pipeline/#db-aggregate-stages>` which does not require an underlying collection. For aggregations on collection data, see :phpmethod:`MongoDB\\Collection::aggregate()`.
```

The manual doesn't do so, but I think we can link "admin/diagnostic pipeline" to https://docs.mongodb.com/manual/reference/operator/aggregation-pipeline/#db-aggregate-stages.